src/Jackett.Common/Indexers/TorrentsCSV.cs

```diff
@@ -14,77 +14,63 @@
 
 namespace Jackett.Common.Indexers
 {
-    public class Torrentscsv : BaseWebIndexer
+    public class TorrentsCSV : BaseWebIndexer
     {
-
         private string ApiEndpoint => SiteLink + "service/search";
 
-        private new ConfigurationData configData
-        {
-            get => base.configData;
-            set => base.configData = value;
-        }
-
-        public Torrentscsv(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps) : base(
-            name: "Torrents.csv",
-            description: "Torrents.csv is a self-hostable, open source torrent search engine and database",
-            link: "https://torrents-csv.ml/",
-            caps: new TorznabCapabilities
-            {
-                SupportsImdbMovieSearch = true
-            },
-            configService: configService,
-            client: wc,
-            logger: l,
-            p: ps,
-            configData: new ConfigurationData())
+        private new ConfigurationData configData => base.configData;
+
+        public TorrentsCSV(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
+            : base("Torrents.csv",
+                   description: "Torrents.csv is a self-hostable, open source torrent search engine and database",
+                   link: "https://torrents-csv.ml/",
+                   caps: new TorznabCapabilities
+                   {
+                       SupportsImdbMovieSearch = true
+                   },
+                   configService: configService,
+                   client: wc,
+                   logger: l,
+                   p: ps,
+                   configData: new ConfigurationData())
         {
             Encoding = Encoding.UTF8;
             Language = "en-us";
             Type = "public";
 
-            // dummy mappings for sonarr, radarr, etc since torrents.csv doesnt return categories
```

Comment:
This comment suggests that Sonarr and Radarr won't use this tracker for searching without these mappings. Is this not the case?

Reply:
It will work but you have to change the category in Sonarr/Radarr to other. We have a lot of indexers with category Other only and they are fine.